cypress/e2e/nonParallel/data-import/e2e/match-on-pol-and-update-instance.cy.js

```diff
@@ -63,7 +63,10 @@ describe('ui-data-import', () => {
     },
     {
       mappingProfile: { typeValue: NewFieldMappingProfile.folioRecordTypeValue.item,
-        name: mappingProfileNameForItem },
+        name: mappingProfileNameForItem,
+        status:'Available',
+        permanentLoanType:'"Can circulate"',
+        materialType:'"book"' },
```

Comment:
Let's use here  constants from cypress\support\constants.js
and add one value to item statuses : 'Available'

Reply:
use ITEM_STATUSES.AVAILABLE, LOAN_TYPE_NAMES.CAN_CIRCULATE, MATERIAL_TYPE_NAMES.BOOK

Reply:
I think it's be better to create an object with properties and values instead constant, because item has many statuses, many loan types, material types.